### ISSUE TITLE
Update dependencies

### ISSUE THREAD

**Issue by jpmcfarland** [rijksuniversiteit-groningen/iBridges-Gui#26]
Update to the most recent dependencies practical (e.g., Python iRODS client)

**Comment by jpmcfarland** [comment]
The PRC is already at it latest version.  Check the other dependencies for updates.  Check how much work converting to PyQt6 will be.

**Comment by jpmcfarland** [comment]
PyQt6 upgrade was actually simple.  This `sed` script did the bulk of the work:

```sed
s/exec_/exec/g
s/PyQt5/PyQt6/g
s/QtWidgets.QFileSystemModel/QtGui.QFileSystemModel/g
s/Qt.Checked/Qt.CheckState.Checked/g
s/Qt.Unchecked/Qt.CheckState.Unchecked/g
s/Qt.CheckStateRole/Qt.ItemDataRole.CheckStateRole/g
s/Qt.DisplayRole/Qt.ItemDataRole.DisplayRole/g
s/Qt.EditRole/Qt.ItemDataRole.EditRole/g
s/Qt.ItemIsUserCheckable/Qt.ItemFlag.ItemIsUserCheckable/g
s/Qt.ItemIsAutoTristate/Qt.ItemFlag.ItemIsAutoTristate/g
s/QMessageBox.No/QMessageBox.StandardButton.No/g
s/QMessageBox.Yes/QMessageBox.StandardButton.Yes/g
s/QtCore.Qt.WaitCursor/QtCore.Qt.CursorShape.WaitCursor/g
s/QtCore.Qt.ArrowCursor/QtCore.Qt.CursorShape.ArrowCursor/g
s/QHeaderView.ResizeToContents/QHeaderView.ResizeMode.ResizeToContents/g
s/QtCore.QStandardPaths.HomeLocation/QtCore.QStandardPaths.StandardLocation.HomeLocation/g
s/QLineEdit.Password/QLineEdit.EchoMode.Password/g
s/QtCore.Qt.ItemIsEnabled/QtCore.Qt.ItemFlag.ItemIsEnabled/g
```

**Comment by jpmcfarland** [comment]
Resolved by pull request #45.
